builder/src/Build.hs

```diff
@@ -952,7 +952,7 @@ crawlRoot env@(Env _ _ projectType _ buildID _ _) mvar root =
           Right modul@(Src.Module _ _ _ imports values _ _ _ _) ->
             do
               let deps = map Src.getImportName imports
-              let local = Details.Local path time deps (any isMain values) buildID buildID
+              let local = Details.Local path time deps (any (isMain . snd) values) buildID buildID
```

Comment:
This goes along with https://github.com/gren-lang/compiler/pull/117#discussion_r961431270

I think I'll switch it to `(isMain . snd)` in all places (and avoid the `fmap snd` alternative) since that seems like it'd be easier for ghc to optimize.